docs/markdown/DevGuide_BuildProcessor.md

```diff
@@ -0,0 +1,53 @@
+# Build your own Processor
+This guide will walk you through the creation of a new processor.
+
+## Necessary files and changes
+The easiest way to start creating a new processor in Inviwo is by using Inviwo-meta (TODO: link to meta docs).
+In order to create a new processor in Inviwo, you need to create the source files for the processor itself, register it in the according module and add it to the Cmake files.
+Inviwo-meta does all of those things for you automatically. The following describes the files that need to be created or modified:
+1. You need to create the processors source and header files in `<module-dir>/src/processors/`
+2. The processor needs to be registered in its module. To do so, open the `<module-dir>/src/<module-name>module.cpp` and include your processor's header, then add the following line to the module's constructor:
+`registerProcessor<MyNewProcessor>();`
+3. Add your new processor's header and source to the module's `CMakeLists.txt` (`<module-dir>/CMakeLists.txt`)
+
+
+## Processor Subclassing
+If you have created your new processor using Inviwo-meta, you can see there is already a class definition in your created files.
+Your processor needs to be a subclass of `Processor` and it needs to override the `void Processor::process()` method. The `process()` method is where a processors action is implemented.
+Additionally, you can override the `const ProcessorInfo getProcessorInfo() const` method which returns meta information about the processor, like a class identifier, name, category, tags etc.
```

Comment:
mention URI

Reply:
what do you mean by that?

Reply:
I am referring to the class identifier, which needs to be unique (and is in URI style). In addition, the processor template already provides a `getProcessorInfo()` function, which can/should be modified. 

```c++
const ProcessorInfo VolumeRaycaster::processorInfo_{
    "org.inviwo.VolumeRaycaster",  // Class identifier
    "Volume Raycaster",            // Display name
    "Volume Rendering",            // Category
    CodeState::Stable,             // Code state
    "GL, DVR, Raycasting"          // Tags
};
```